public/app/plugins/panel/table/migrations.ts

```diff
@@ -16,6 +28,194 @@ export const tableMigrationHandler = (panel: PanelModel<Options>): Partial<Optio
   return panel.options;
 };
 
+const transformsMap = {
+  timeseries_to_rows: 'seriesToRows',
+  timeseries_to_columns: 'seriesToColumns',
+  timeseries_aggregations: 'reduce',
+};
```

Comment:
I think we might need to add `table -> merge` in this scenario.

Reply:
Thanks for spotting this @mckn. Have added it to the mapping.